cranelift-entity/src/map.rs

```diff
@@ -56,11 +56,21 @@ where
         }
     }
 
+    /// Returns the number of elements in the underlying vector.
+    pub fn len(&self) -> usize {
+        self.elems.len()
+    }
```

Comment:
Instead of using `.len()`, can the code that would use this use `.keys().size_hint()` instead?

Reply:
Well, the exact length should be known for the serializer API. But I'll check whether currently used serializer requires the size up front.

Reply:
It's not required. I have removed the `.len()` method.

Reply:
I haven't noticed it, since cache system fails quietly. The size information is required.

Reply:
Ok, then let's go ahead and add `len()` back then. But please add a comment mentioning that the length of a `SecondaryMap` is not necessarily the same as the length of the corresponding `PrimaryMap`.